pure_pursuit_wrapper/launch/pure_pursuit_wrapper.launch

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0"?>
+<launch>
+    <!-- Pure Pursuit Node -->
+    <include file="$(find waypoint_follower)/launch/pure_pursuit.launch">
+        <arg name="is_linear_interpolation" value="True"/>
+        <arg name="publishes_for_steering_robot" value="True"/>
+    </include>
+    <!-- Pure Pursuit Wrapper Node -->
+    <node pkg="pure_pursuit_wrapper" type="pure_pursuit_wrapper_node" name="pure_pursuit_wrapper_node" output="screen">
+        <rosparam command="load" file="$(find pure_pursuit_wrapper)/config/default.yaml" />
+    </node>
+
+    <!-- pure_pursuit -->
```

Comment:
Rather than commenting this out you should make a pure_pursuit_wrapper_tester.launch file with a comment explaining what this does.

Reply:
fixed